src/Compatibility/Microsoft.DotNet.Compatibility/ValidatePackage.cs

```diff
@@ -18,35 +18,35 @@ namespace Microsoft.DotNet.Compatibility
     public class ValidatePackage : TaskBase
     {
         [Required]
-        public string PackageTargetPath { get; set; }
+        public string? PackageTargetPath { get; set; }
```

Comment:
These Required parameters should not be marked as nullable

Reply:
As mentioned below, it must be marked as nullable as the value doesn't get set in the constructor (it's an msbuild task).